operator/helper/multiline.go

```diff
@@ -33,7 +33,7 @@ type FlusherConfig struct {
 func NewFlusherConfig() FlusherConfig {
 	return FlusherConfig{
 		// Empty or `0s` means that we will never force flush
-		Period: Duration{Duration: 0},
+		Period: Duration{Duration: time.Hour},
```

Comment:
Per the suggestion [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/4849#issuecomment-911926696), the default should be a few hundred milliseconds.

Since the default `poll_interval` is `200ms`, I suggest we use `500ms` here. This will ensure that at least two subsequent polls have occurred before flushing an incomplete log.

Also, we need to update the docs accordingly.

Reply:
whoops, must of misread and ok updated the doc as well.